### PR TITLE
Add hmrc-frontend-scaffold assets to project

### PR DESCRIPTION
**Pull request by amaalali**

### CODE REVIEW

File: conf/prod.routes [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Add all the application routes to the app.routes file
-->         /transit-movements-trader-admin-frontend              app.Routes
```
Comment (ericohwotu) [inline conf/prod.routes:2]: admin route? or will this be on a seperate pr?

File: conf/logback.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <configuration>
 
     <appender name="FILE" class="ch.qos.logback.core.FileAppender">
-        <file>logs/transit-movements-trader-admin-frontend.log</file>
```
Comment (ericohwotu) [inline conf/logback.xml:5]: name

File: conf/prod.routes (hunk 1 above)
Comment (amaalali) [reply conf/prod.routes:2]: I was planning to do it later. I was thinking the base service url should be similar, ie `/manage-transit-movements/administration-dashboard`

**Review by ericohwotu — APPROVED** [review]

File: conf/prod.routes (hunk 1 above)
Comment (amaalali) [reply conf/prod.routes:2]: I've updated that now

File: conf/logback.xml (hunk 2 above)
Comment (amaalali) [reply conf/logback.xml]: I've reverted that now

**Pull request merged**